ch04.adoc

```diff
@@ -210,304 +226,208 @@ double time(time) ;
+Leap seconds should be considered when processing datetimes and time-coordinates in the **`standard`** and **`utc`** calendars, which treat them differently.
+In the other CF calendars, leap seconds never need to be considered.
 
-In the **`julian`** and the default **`standard`** calendar, dates in years before year 0 (i.e. before 0-1-1 0:0:0) are not allowed, and the year in the reference datetime of the units must not be negative.
-In these calendars, year zero has a special use to indicate a climatology (see <<climatological-statistics>>), but this use of year zero is deprecated.
-In other calendars, year 0 is the year before year 1, and negative years are allowed.
+For observational or model-generated data that follows the Gregorian calendar and UTC (after 1972) including leap seconds, if precision to the second is important, the **`utc`** calendar is recommended in preference to the **`standard`** calendar.
+For model-generated data following the Gregorian calendar without leap seconds, the **`proleptic_gregorian`** calendar is recommended in preference to the **`standard`** calendar.
+See also <<leap-seconds>>.
 
-**`standard`**:: Mixed Gregorian/Julian calendar as defined by UDUNITS.
-This is the default.
+The possible values for the **`calendar`** attribute and the definitions of the calendars are as follows:
+
+**`standard`**:: The Gregorian--Julian calendar as defined by UDUNITS.
+This is the default calendar when there is no **`calendar`** attribute, except when the attribute `month_lengths` is present (see <<explicit-calendar>>).
 A deprecated alternative name for this calendar is **`gregorian`**.
 The Gregorian and Julian calendars have the same lengths of their months; they differ only in respect of the rules that decide which years are leap years.
 In the **`standard`** calendar, datetimes after and including 1582-10-15 0:0:0 are in the Gregorian calendar, in which a year is a leap year if either (i) it is divisible by 4 but not by 100 or (ii) it is divisible by 400.
 Datetimes before (and excluding) 1582-10-5 0:0:0 are in the Julian calendar, in which any year that is divisible by 4 is a leap year.
 Year 1 AD or CE in the **`standard`** calendar is also year 1 of the **`julian`** calendar.
 Negative years are invalid in time coordinates and reference datetimes in the **`standard`** calendar.
-In the **`standard`** calendar, 1582-10-15 0:0:0 is exactly 1 day later than 1582-10-4 0:0:0.
-Therefore datetimes in the range from (and including) 1582-10-5 0:0:0 until (but excluding) 1582-10-15 0:0:0 are invalid, and must not be used as reference in **`units`**.
++
+In the **`standard`** calendar, datetimes in the range from (and including) 1582-10-5 0:0:0 until (but excluding) 1582-10-15 0:0:0 are invalid.
+Datetimes in this range must not be used as reference in **`units`**, and the interval between the datetimes 1582-10-15 0:0:0 and 1582-10-4 0:0:0 is exactly 1 day.
```

Comment:
I was confused for a long moment by the fact that 10-15 is first and 10-04 is second.  I would suggest swapping them to the natural order: `interval between the datetimes 1582-10-4 0:0:0 and 1582-10-15 0:0:0`

Reply:
Done. I can't remember why it was backwards!